### DISCUSSION TITLE
Disbox Via FTP

### DISCUSSION THREAD

**Discussion by tris203** [DisboxApp/web#62] [Show and tell]
Hi all

Just letting you know about something I have been working on. 

Got a very basic implementation working of Disbox storage via FTP

It uses ftp-srv with an extended VFS for Disbox

https://github.com/tris203/disftp

At the moment it essentially uses the library as is, with a few modifications for parsing buffers instead of blobs

Thanks for the awesome classes, it was really fun getting in depth with them and working out how it all hangs together 
@DisboxApp

**Comment by DisboxApp** [comment]
This is really cool!
Tried running it locally and connect using WinSCP and for me it seems to have some errors, but I read the code and it looks awesome.

Curious to see it's possible to use the untested `moveFile` and `updateFile` to implement more functionality.

**Reply by tris203** [reply]
What errors were you getting?

I've just finished rewriting the whole thing in Typescript (Including the core Disbox classes)

Next is to package it up into a Docker Container so it could be self hosted easily
